recipes/libgsasl/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rm, rmdir
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.52.0"
```

Comment:
```suggestion
required_conan_version = ">=1.50.0"
```
1.51.3 more than enough without export_conandata_patches